pkg/api/types.go

```diff
@@ -1469,8 +1469,10 @@ type NodeSystemInfo struct {
 
 // NodeStatus is information about the current status of a node.
 type NodeStatus struct {
-	// Capacity represents the available resources of a node.
+	// Capacity represents the total resources of a node.
 	Capacity ResourceList `json:"capacity,omitempty"`
+	// Allocatable represents the resources of a node that are available for scheduling.
+	Allocatable ResourceList `json:"allocatable,omitempty"`
```

Comment:
I'd prefer we don't make that assumption, unless there's a specific requirement. I suppose we could make the weak assertion that "Changes are relatively infrequent". But I think that by having it in status (and not spec), we're suggesting that it can change.